UPGRADE.md

```diff
@@ -101,6 +101,16 @@ urls = array(
     'es' => '/'
 );
 ```
+### Texteditor Content Type params changed
+
+| Before                                        | Now                                             |
+|-----------------------------------------------|-------------------------------------------------|
+| `<param name="tables" value="true" />`        | `<param name="table" value="true" />`           |
+| `<param name="links" value="true" />`         | `<param name="link" value="true" />`            |
+| `<param name="pasteFromWord" value="true" />` | `<param name="paste_from_word" value="true" />` |
+| `<param name="maxHeight" value="500" />`      | `<param name="max_height" value="500" />`       |
+| `<param name="iframes" value="true" />`       | allowed as default                              |
+| `<param name="scripts" value="true" />`       | allowed as default                              |
```

Comment:
You should probably also add that these two params are replaced by the allowed content parameter (however that one is called). And please add a line of explanation before the table.

And for consistency the title should not be an entire sentence, so write something like `Texteditor`.